unit-tests/src/test/java/org/eclipse/collections/impl/list/IntervalTest.java

```diff
@@ -53,6 +53,21 @@ public void into()
         Assert.assertEquals(10, sum);
     }
 
+    @Test
+    public void fromUntilTo()
+    {
+        Interval interval = Interval.from(1);
```

Comment:
Please add tests for:
1)  negative numbers
2) positive to negative numbers
3) negative to positive numbers
4) Integer.MIN to Integer.MAX
5) Integer.MAX to Integer.MIN
6) Tests to show parity with IntStream().range()